tests/typings/Snapshot.test-d.ts

```diff
@@ -0,0 +1,107 @@
+import type { TypeEqual } from 'ts-expect'
+import { expectType } from 'ts-expect'
+import { INTERNAL_Snapshot as Snapshot } from '../../src/vanilla'
```

Comment:
I think this breaks the old ts tests.

Reply:
```suggestion
import { INTERNAL_Snapshot as Snapshot } from 'valtio/vanilla'
```
might help.

Also, I'm not sure if `tests/*/*.ts` is supported. Suppose it does, can you name the file `tests/typings/snapshot.test.ts`?

Reply:
I'm not sure if I understand you correctly. These files tests as regular `tsc --noEmit`, not via `jest`. According to [`tsconfig.json`](https://github.com/pmndrs/valtio/blob/e159c821672c6e3914346131fe2aec64dad8f35f/tsconfig.json#L17), `tsc` should go through this file as expected. Why do I need to rename it to `*.test.ts`? Also, if I do so, I have to ignore it in jest.